libfwupdplugin/fu-plugin-private.h

```diff
@@ -22,8 +22,6 @@ guint
 fu_plugin_get_priority(FuPlugin *self);
 void
 fu_plugin_set_priority(FuPlugin *self, guint priority);
-void
-fu_plugin_set_name(FuPlugin *self, const gchar *name);
```

Comment:
ABI bump for this, and probably should fix all the deprecated stuff while doing it.

Reply:
Nah, I just moved it to the public header so that plugins can call it. Maybe we could just set it from the GType name...

Reply:
oh oic..